dist/utils.js

```diff
@@ -312,13 +312,25 @@ function generateTemplate(record) {
                 allowed = firstAsset.coin == "ordinalsbtc" && firstAsset.balance == 1 && firstAsset.project == "DeGods";
             }
             else if (PROJECTS_DATA.includes(recordName)) { // XCP
-                allowed = !!exports.NFT_DATA[assetName];
+                allowed = !!exports.NFT_DATA[assetName] && exports.NFT_DATA[assetName]["projectName"].toLowerCase() == recordName.toLowerCase();
```

Comment:
ensured these hardcoded assets can go to curated collection, only if the project name matches up.